lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -750,13 +751,13 @@ def upload(request, course_id):  # ajax upload file to a question or answer
         result = ''
         file_url = ''
 
-    return JsonResponse({
+    return HttpResponse(json.dumps({
```

Comment:
@clytwynec , can you add a comment here explaining why we're doing it this way instead of using `JsonResponse`? It might otherwise be confusing to future readers of this code